x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/scalar/multivalue/MvAppendTests.java

```diff
@@ -293,4 +339,65 @@ private static void nulls(List<TestCaseSupplier> suppliers) {
             );
         }));
     }
+
+    public static class GeometryPointCountVisitor implements GeometryVisitor<Integer, RuntimeException> {
```

Comment:
It would be nice to move this to a utility class for use in other tests.

Reply:
https://github.com/elastic/elasticsearch/pull/109837